lms/djangoapps/ccx/tests/test_views.py

```diff
@@ -124,6 +123,75 @@ def setup_students_and_grades(context):
                     )
 
 
+class TestAdminAccessCoachDashboard(CcxTestCase, LoginEnrollmentTestCase):
```

Comment:
Is there a test in here or somewhere else that tests for the not user case here: https://github.com/mitocw/edx-platform/blob/feature/aq/allow_staff_view_coach_dashboard_mitocw%23125/lms/djangoapps/ccx/plugins.py#L28